manuscript.md

```diff
@@ -42,7 +46,35 @@
 
 ## How are interaction strengths actually inferred? 
 
-## Could we use hypergraphs and multi-layer networks to predict more interactions? 
+## Could we use hypergraphs and multi-layer networks to predict more interactions?
+
+The web of life is built by a variety of interactions that can vary over time
+and space; these interactions include facilitation, niche construction,
+zoonoses, vector-borne diseases, among others [@Garcia-Callejas2018MulInt],
+which all share the fact that interactions between species are themselves
+interacting. One mathematical tool to describe these situations is hypergraphs:
+hypergraphs are the generalization of a graph, allowing a broad and manageable
+approach to complex interactions [@Carletti2020DynSys]. @Golubski2011ModMod were
+among the first to show that interaction modifiers are themselves interacting,
+which makes the complexity of ecological networks explode. Investigating
+hypergraphical interactions can be more important than previously thought, as
+they can reveal the importance of species within a network based on *indirect*
+interactions [@Golubski2016EcoNet]. An additional degree of complexity is
+introduced by multi-layer networks [@Hutchinson2019SeeFor]. Multi-layer networks
+offer links across "variants" of the networks, which can be patches, timepoints,
+environments, etc. As @Pilosof2017MulNat suggest, ecological networks are
+intrinsically multi-layers, as they are shaped by evolution, dispersal,
```

Comment:
I'm not to clued up on terminology not sure if grammar errors or not - should it maybe be: 'networks are intrinsically *multi-layered*'?

Reply:
i'm also fond of [this paper](https://conbio.onlinelibrary.wiley.com/doi/pdf/10.1111/cobi.12943) if we have room for an extra citation

Reply:
Both fixed in the next commit.